bubble/src/main/java/pl/touk/android/bubble/Degree.kt

```diff
@@ -3,5 +3,6 @@ package pl.touk.android.bubble
 internal class Degree {
     companion object Degree {
         val MINUS_45 = (-Math.PI/4).toFloat()
+        val PLUS_45 = (Math.PI/4).toFloat()
```

Comment:
Good point but unfortunately I have managed to merge PR :) thus it will have to wait for new PR.